src/main.rs

```diff
@@ -1,14 +1,46 @@
-use std::path::PathBuf;
+use std::{io::Write, path::PathBuf};
```

Comment:
Not a problem but just noting for @jntrnr and others: how do we want to handle multi-layered use's? There's a rustfmt config but I can't remember whether it's stable yet, and there are arguments against and for both approaches.

Reply:
I lean towards just using the default `rustfmt` style